### DISCUSSION TITLE
gatsby develop not working as expected

### DISCUSSION THREAD

**Discussion by SohaWithAnS** [gatsbyjs/gatsby#36655] [Help] [answered]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

I wish to utilize gatsby to make my own portfolio website. I cloned an existing gatsby starter on GitHub (https://github.com/margueriteroth/gatsby-prismic-starter-prist) and followed the steps as they are given. 

After I have created my site using Gatsby CLI , and done `cd my-project` , `npm install` gives me a lot of warnings and vulnerability reports. I went through previous (now closed) issues about this under issues, and I understand that while the warnings are scary, there's nothing to fix as it's not relevant to Gatsby. I did `npm audit fix` and ignored the rest of the vulnerability reports. 

Now, when I do `gatsby develop`, it does not work as expected and results in a "not finished" at the end. I've attached my terminal output when I run `gatsby develop`. 

<img width="787" alt="Screen Shot 2022-09-21 at 1 55 00 AM" src="https://user-images.githubusercontent.com/36818003/191438608-b3b7ede2-51f6-4026-aef0-816f7022d3e5.png">
<img width="787" alt="Screen Shot 2022-09-21 at 1 55 17 AM" src="https://user-images.githubusercontent.com/36818003/191438658-e8c11e8a-2acf-47e7-9bbd-b6883eec402c.png">
<img width="787" alt="Screen Shot 2022-09-21 at 1 55 34 AM" src="https://user-images.githubusercontent.com/36818003/191438705-8ddb6b6a-5f4a-4e32-bd36-c021e4e5304c.png">

I also went through the Gatsby CLI commands and made sure I was doing it right, and it seems right, as per the official documentation - 
<img width="1376" alt="Screen Shot 2022-09-21 at 1 46 31 AM" src="https://user-images.githubusercontent.com/36818003/191439423-25fc82a0-5178-425c-a45a-bc974f893b32.png">

I'm unable to see the starter template at localhost:8000 (which makes sense because the output of `gatsby develop` ends with 'not finished'). I'm not sure how to fix this. Any help would be appreciated. Thank you. 

### Reproduction Link

https://codesandbox.io/s/github/gatsbyjs/gatsby-starter-minimal

### Steps to Reproduce

1. cd PROJECT-NAME
2. npm install
3. gatsby develop

### Expected Result

The expected result is a successful run of `gatsby develop` so I can see my site at localhost:8000. 

<img width="1007" alt="Screen Shot 2022-09-21 at 2 22 32 AM" src="https://user-images.githubusercontent.com/36818003/191440622-f850a571-e469-4a05-9979-8b9c928bb800.png">

### Actual Result

`gatsby develop` not working properly. Unable to see site at localhost:8000. 

### Environment

```shell
System:
    OS: macOS 12.5.1
    CPU: (8) x64 Apple M1
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 14.18.1 - /usr/local/bin/node
    npm: 6.14.15 - /usr/local/bin/npm
  Browsers:
    Chrome: 105.0.5195.125
    Safari: 15.6.1
  npmPackages:
    gatsby: ^4.23.0 => 4.23.0
    gatsby-image: ^2.2.20 => 2.11.0
    gatsby-plugin-catch-links: ^2.0.12 => 2.10.0
    gatsby-plugin-emotion: ^4.1.0 => 4.5.0
    gatsby-plugin-google-analytics: ^2.1.6 => 2.11.0
    gatsby-plugin-manifest: ^4.23.0 => 4.23.0
    gatsby-plugin-offline: ^3.0.8 => 3.10.2
    gatsby-plugin-react-helmet: ^3.1.8 => 3.10.0
    gatsby-plugin-remove-trailing-slashes: ^2.0.11 => 2.10.0
    gatsby-plugin-resolve-src: ^2.0.0 => 2.1.0
    gatsby-plugin-sass: ^2.0.11 => 2.8.0
    gatsby-plugin-sharp: ^4.23.0 => 4.23.0
    gatsby-source-filesystem: ^4.23.0 => 4.23.0
    gatsby-source-prismic-graphql: 3.4.0-beta.1 => 3.4.0-beta.1
    gatsby-transformer-sharp: ^2.2.16 => 2.12.1
  npmGlobalPackages:
    gatsby-cli: 4.23.0
```

**Comment by LekoArts** [comment, marked as answer]
Hi!

You're running into this (https://github.com/margueriteroth/gatsby-prismic-starter-prist/issues/24) because the starter itself is severely outdated. If you want to create a Gatsby + Prismic project I'd strongly advise to follow https://prismic.io/docs/technologies/gatsby

If you'd absolutely want to use the linked starter you'd have to update from Gatsby 2 to 4 and also migrate from an older Prismic plugin to the newer one, which is all quite a lot of work.

Starters that work with the latest version of Gatsby are listed here: https://www.gatsbyjs.com/starters/

**Reply by SohaWithAnS** [reply]
Thank you for your response! I will check out the starters that work with the latest version of Gatsby.
